docs/book/features/volume_expansion.md

```diff
@@ -1,6 +1,16 @@
-# vSphere CSI Driver - Offline Volume Expansion
+# vSphere CSI Driver - Volume Expansion
 
-CSI Volume Expansion was introduced as an alpha feature in Kubernetes 1.14 and it was promoted to beta in Kubernetes 1.16. The vSphere CSI driver currently extends this support for dynamically/statically created offline block volumes only i.e allows a block volume to be extended when it is not attached to a node. Check the [supported features](../supported_features_matrix.md) section to verify if your environment conforms to all the required versions and the [known issues](../known_issues.md) section to see if this feature caters to your requirement. Note that offline volume expansion is available from vSphere CSI v2.0 onwards in Vanilla Kubernetes and v2.1 onwards in Tanzu Kubernetes Grid Service (TKGS). Volume expansion is currently not supported in the Supervisor cluster.
+CSI Volume Expansion was introduced as an alpha feature in Kubernetes 1.14 and it was promoted to beta in Kubernetes 1.16. The vSphere CSI driver supports volume expansion for dynamically/statically created **block** volumes only. Kubernetes supports two modes of volume expansion - offline and online. When the PVC is being used by a Pod i.e it is mounted on a node, the resulting volume expansion operation is termed as an online expansion. In all other cases, it is an offline expansion. Depending upon the kubernetes flavor and the mode of volume expansion required for your use case, refer to the table below to know the minimum version of the vSphere CSI driver to be used.
```

Comment:
Please add some notes to explain what you need to do for volume expansion to work for a "statically created **block** volume".  By default, this is not supported in Kubernetes.  Admin will have to make sure a storageClassName that allows volume expansion is in the PV spec to work around it.

Reply:
Added the following in `Expand PVC` section:

If you are using a statically created PVC, ensure that the PVC and the PV specs have the `storageClassName` parameter pointing to a storage class which has `allowVolumeExpansion` set to true.